modular_zubbers/code/modules/jobs/_job.dm

```diff
@@ -24,7 +24,6 @@
 
 /datum/job/research_director
 	required_character_age = 21
```

Comment:
So why is RD just 21?

Reply:
Leftover from when it was 21 and 25, I've been nudging it back and forth, current idea seems to be 21 is preferred so there'll be a commit for that

Reply:
21/25 seems like a good middle ground